SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -93,33 +110,78 @@ public async Task<IActionResult> UpdateUser(int id, [FromBody] User user)
                         await _context.SaveChangesAsync();
                         return Ok("User update successful");
                     }
-
                 }
             }
             else
             {
                 return new BadRequestObjectResult("user cannot be null");
             }
         }
+
         // POST: api/user/login
+        [AllowAnonymous]
         [HttpPost("login")]
-        public async Task<ActionResult> LoginPost([FromBody]User user)
+        public IActionResult CreateToken([FromBody]User user)
+        {
+            IActionResult response = Unauthorized();
+            var _user = Authenticate(user);
+            
+
```

Comment:
You can remove these trailing type of spaces.

Reply:
I removed the spaces trailing